ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const PERMISSION_TYPES = {
+  eth_accounts: {
+    leftIcon: 'fas fa-eye',
+    label: 'eth_accounts',
+    rightIcon: null,
+  },
+};
+
+export default function PermissionsConnectPermissionList({ permissions }) {
+  const t = useI18nContext();
+  return (
+    <div className="permissions-connect-permission-list">
+      {Object.keys(permissions)
+        .map((permission) => PERMISSION_TYPES[permission])
+        .map((permission) => (
+          <div className="permission" key={permission.label}>
+            <i className={permission.leftIcon} />
+            {t(permission.label)}
+            <i className={permission.rightIcon} />
```

Comment:
Should this be conditional if it's not required? Will align text in layout better?
```suggestion
            {Boolean(permission.rightIcon) && <i className={permission.rightIcon} />}
```

Reply:
It's not conditional to adjust to other permissions that may have icons on the right. Otherwise the text will flow in place where there might be an icon